src/overlays/actors/ovl_Item_Shield/z_item_shield.c

```diff
@@ -62,24 +65,24 @@ void ItemShield_Init(Actor* thisx, PlayState* play) {
     s32 i;
 
     this->timer = 0;
-    this->unk_19C = 0;
+    this->stateFlags = 0;
 
     switch (this->actor.params) {
-        case 0:
+        case ITEMSHIELD_TYPE_COLLECTIBLE:
             ActorShape_Init(&this->actor.shape, 1400.0f, NULL, 0.0f);
             this->actor.shape.rot.x = 0x4000;
-            ItemShield_SetupAction(this, func_80B86BC8);
+            ItemShield_SetupAction(this, ItemShield_Collectable_Wait);
```

Comment:
collectible/collectable probably should be consistent?

Reply:
Does anyone care which? Apparently both are valid.

Reply:
to me "collectible" is the noun, the thing that is collectable (collect-able)
thats prob just something ive made up in my head and not actually standard though lol

Reply:
"collectable" is only an adjective so probably go with "collectible"?

Reply:
OED only lists "-able", with the note
> Forms:  Also (now chiefly U.S.) -ible.

Reply:
Although interestingly the noun form apparently originates from the US.

Reply:
Well Item00 and elsewhere uses "ible", so I'll use that for the sake of standardisation.